plugins/chat/app/services/chat/create_direct_message_channel.rb

```diff
@@ -90,9 +89,5 @@ def update_memberships(guardian:, channel:, target_users:, **)
         unique_by: %i[user_id chat_channel_id],
       )
     end
-
-    def publish_channel(channel:, target_users:, **)
-      Chat::Publisher.publish_new_channel(channel, target_users)
-    end
```

Comment:
This one is important. It prevents green dot to show when you create a direct message channel. We don't notify until a message is sent, otherwise we might just show a channel with no content.